src/popup/components/PanelItem.vue

```diff
@@ -56,36 +56,22 @@ export default {
 @use '../../styles/typography';
 
 .panel-item {
-  margin: 8px;
-  padding: 8px 2px 8px 12px;
-  width: auto;
-  min-height: 48px;
-  border-radius: 10px;
+  @extend %face-sans-15-regular;
+
   display: flex;
-  justify-content: space-between;
-  flex-direction: row;
   align-items: center;
+  width: 100%;
+  min-height: 48px;
+  margin-bottom: 8px;
+  padding: 8px 2px 8px 12px;
+  border-radius: variables.$border-radius-interactive;
+  justify-content: space-between;
   text-decoration: none;
   text-align: left;
-  background: rgba(variables.$color-white, 0.08);
-  color: rgba(variables.$color-white, 0.95);
-  -webkit-transition: 0.4s;
-  transition: 0.4s;
-  opacity: 0.75;
+  color: rgba(variables.$color-white, 0.75);
+  transition: variables.$transition-interactive;
```

Comment:
If you touched imports refactoring in that PR maybe you could also remove `variables.` here?

Reply:
OK, done.